mythril/laser/smt/solver/solver.py

```diff
@@ -52,7 +55,12 @@ def check(self, *args) -> z3.CheckSatResult:
         """
         old_stdout = sys.stdout
         sys.stdout = open(os.devnull, "w")
-        evaluate = self.raw.check(args)
+        try:
+            evaluate = self.raw.check(args)
+        except z3.z3types.Z3Exception as e:
+            # Some requests crash the solver
+            evaluate = None
```

Comment:
Should we adapt the function signature to `Optional[z3.CheckSatResult]`?

Reply:
Now that I see it might be better if we use z3.unknown over None